plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -77,6 +77,9 @@ var resourceToResourceName = map[unversioned.GroupResource]api.ResourceName{
 	api.Resource("resourcequotas"):         api.ResourceQuotas,
 	api.Resource("secrets"):                api.ResourceSecrets,
 	api.Resource("persistentvolumeclaims"): api.ResourcePersistentVolumeClaims,
+	extensions.Resource("replicasets"):     extensions.ResourceReplicaSets,
```

Comment:
I think we want everything a normal user could create here, or anything a normal user might reasonably be expected to create.